src/NServiceBus.Core/Sagas/Sagas.cs

```diff
@@ -37,6 +37,9 @@ internal Sagas()
         /// </summary>
         protected internal override void Setup(FeatureConfigurationContext context)
         {
+            context.Pipeline.Register("PopulateAutoCorrelationHeadersForReplies", new PopulateAutoCorrelationHeadersForRepliesBehavior(),
```

Comment:
doesn't this behavior also forward saga ids across endpoints which don't even use sagas (and which have the saga feature itself disabled)? In that case we would break this because endpoints without sagas wouldn't forward that header anymore?

Reply:
I don't know. Raised the question about correlation ID and what got back was that AutoCorrelation ID is only used in conjunction with Sagas. @andreasohlund, could you help with this?